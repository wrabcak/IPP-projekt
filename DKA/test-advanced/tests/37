# epsilon uzaver
({s1, s2, s3, s4, s5},
{'a', 'b', 'c'},
{
s1 'a' -> s3,
s1 'b' -> s2,

s2 'c' -> s5,

s3 'b' -> s4,
s3 'c' -> s2,

s4 'a' ->s4

},
s1
, {s4, s5} )
