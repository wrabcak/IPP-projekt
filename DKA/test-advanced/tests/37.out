(
{qFALSE, s1, s2, s3, s4, s5},
{'a', 'b', 'c'},
{
qFALSE 'a' -> qFALSE,
qFALSE 'b' -> qFALSE,
qFALSE 'c' -> qFALSE,
s1 'a' -> s3,
s1 'b' -> s2,
s1 'c' -> qFALSE,
s2 'a' -> qFALSE,
s2 'b' -> qFALSE,
s2 'c' -> s5,
s3 'a' -> qFALSE,
s3 'b' -> s4,
s3 'c' -> s2,
s4 'a' -> s4,
s4 'b' -> qFALSE,
s4 'c' -> qFALSE,
s5 'a' -> qFALSE,
s5 'b' -> qFALSE,
s5 'c' -> qFALSE
},
s1,
{s4, s5}
)